angularjs-example/README.md

```diff
@@ -1,20 +1,34 @@
 angularjs-example
-========
-
+=============
 ## Setup
 
-```
-npm install -g bower
-npm install
-bower install
-```
+1. This demo runs with local Selenium. 
```

Comment:
@jason0x43 please explain, i didn't read the source code, I just read documentation there: https://theintern.github.io/intern/ 
> 3.3SeleniumTunnel – For loading and managing a local standalone Selenium server